### DISCUSSION TITLE
Mac软件折腾之路 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#160] [Announcements]
https://blog.goalonez.site/blog/Mac%E8%BD%AF%E4%BB%B6%E6%8A%98%E8%85%BE%E4%B9%8B%E8%B7%AF.html 

 Goalonez Blog

**Comment by akfc58** [comment]
收集到几个好用但没见过的收集到几个好用但没见过的app。感谢分享！

**Comment by zzfly256** [comment]
看到了不少不错的软件，支持一下

**Comment by arvakme** [comment]
Bartender 的替代品可以看一下 ice，几乎是可以完全替代 Bartender 了https://github.com/jordanbaird/Ice

**Comment by Goalonez** [comment]
> @miao-lv
> Bartender 的替代品可以看一下 ice，几乎是可以完全替代 Bartender 了https://github.com/jordanbaird/Ice

嗯嗯，已经用了一段时间ice了。更新了下拉栏后，总体来说确实已经可以替代Bartender了，当然偶尔还是会遇到一些小问题，不过影响不大。

**Comment by luxuxl** [comment]
同好一起交流一下, 办公我不用 WPS, 用石墨文档, 经测试, 如果对排版要求高, 不能错那肯定是 Office, 排版要求不高 WPS 和石墨都是一样的, 但是石墨非常非常干净

**Comment by luxuxl** [comment]
Flomo 和那几个不是同一个赛道的🤣

**Comment by luxuxl** [comment]
emmm, 那个浏览器有试用过 Brave 吗, 我之前很迷恋这个, 甚至替换掉 Chrome, 但还是换回来了

**Comment by luxuxl** [comment]
输入法说实话我不太敢用微信, 我是 RIME + 搜狗, 期待博主去折腾一下 RIME 🤣

搜狗的配置同步我非常喜欢, 偏好设置里的改符号有一点限制, 我就直接找到配置文件改了, 结果搜狗也能同步

微信输入法我试了下, 似乎没有符号替换的功能, 而且要跨设备粘贴所有设备都得安装微信输入法, 说实话有点排斥, 体积也很大, 我宁愿用 Macbook 和 iPhone 的跨设备粘贴 (PS: 图片也是可以粘贴的哦, 在手机微信聊天框直接粘贴也可以哦)
微信输入法的跨设备同步好像不支持图片......

**Comment by luxuxl** [comment]
没有折腾 Karabiner 吗🤣

**Comment by luxuxl** [comment]
同步可以试一下 Nutstore 坚果云，除了会在 Home 目录生成两个文件（逼死洁癖，体验很不错）

**Comment by luxuxl** [comment]
纠正一下, 微信输入法复制图片需要分类讨论
1、Finder 中复制图片 -> 手机可以粘贴
2、Mac 系统内置截图至剪贴板 -> 手机无法粘贴
3、Mac 第三方截图工具 (Shottr, 其余未测试) -> 手机可以粘贴

**Comment by ltx666** [comment]
专门登陆来感谢一下，开了一堆标签页挨个看。另外问一下可以说下VScode的插件和配置吗，还有时间机器

**Comment by Goalonez** [comment]
> @ltx666
> 专门登陆来感谢一下，开了一堆标签页挨个看。另外问一下可以说下VScode的插件和配置吗，还有时间机器

我的 VScode 只是拿来当文本编辑器使用，写代码用的 WebStorm,所以也没有装什么特别的插件。
时间机器的话目前是通过极空间NAS来做备份的。

**Comment by cckenaaa** [comment]
可以试试 mise，真全平台版本管理工具

**Comment by Goalonez** [comment]
> @cckenaaa
> 可以试试 mise，真全平台版本管理工具

嗯嗯，前两天刚好看到个帖子也有不少人提到，就去了解了一下。暂时还没装上，后面有机会再尝试
